packages/primitives/table/src/types.ts

```diff
@@ -1,8 +1,22 @@
 export type TTableBorderStyle = {
+  borderColor?: string,
+  borderTopWidth?: number,
+  borderBottomWidth?: number,
+  borderLeftWidth?: number,
+  borderRightWidth?: number,
+  borderStyle?: 'solid' | 'dotted' | 'dashed' | 'none',
+}
+
+export type TTableBorderStyleNative = {
```

Comment:
@deepsweet TypeScript was complaining in native table files, because `ViewStyle` doesn't accept "none" as border style. Is it, OK to solve it with duplicating `TTableBorderStyle` like I did here?

Reply:
I think that's because "none" border style is the same as zero width border, so technically `borderStyle: none` is `borderWidth: 0`, and RN decision kinda makes sense. In general we should avoid different types for platforms because from outside it all should have the same API regardless the platform.

One of the ways to solve this is to handle `none` value manually and make it `borderWidth: 0` at the end, avoiding unsupported `borderStyle: none` on native

Reply:
🤔 I wanted to purge all table borders in favor using separate, custom borders on my side, but I think it should work with just setting all the table borders to 0px width. I'll try it out! Thanks!

Reply:
@deepsweet I removed TTableBorderStyleNative, setting border width to 0px worked nicely :) Thanks again! And I cut down the allowed `position` options, so it's ready to merge if you think it's ok